src/Discussion/Discussion.php

```diff
@@ -389,6 +389,16 @@ public function lastUser()
         return $this->belongsTo('Flarum\User\User', 'last_user_id');
     }
 
+    /**
+     * Define the relationship with the discussion's most relevant post.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function mostRelevantPost()
+    {
+        return $this->belongsTo('Flarum\Post\Post', 'most_relevant_post_id');
```

Comment:
So this will only be used when searching, right? Because `most_relevant_post_id` will just be the name of a dynamically calculated column? If so, that should be made clear in the docblock.

Reply:
Yes, good point.